courses/free-courses-en.md

```diff
@@ -410,6 +412,7 @@
 * [Intro To JavaScript Unit Testing & BDD](https://www.youtube.com/watch?v=u5cLK1UrFyQ) - Traversy Media
 * [Javascript course](https://www.youtube.com/playlist?list=PLRAV69dS1uWSxUIk5o3vQY2-_VKsOpXLD) - Hitesh Choudhary
 * [Javascript Essentials](https://www.udemy.com/javascript-essentials/) - Lawrence Turton (Udemy)
+* [Javascript Tutorial for beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7rrvgG7MLNIMSTzVCDZZcT4)
```

Comment:
Following [CONTRIBUTING format guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#formatting):
- Links can't have tracking parameters, so clean it.
- Give attributions to the creator/mentor/platform
- For resource name, use the title as is published in the website
- Add notes about platform registering: ....

Please, apply following snippets to solve it.

```suggestion
* [Javascript Tutorial for Beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7rrvgG7MLNIMSTzVCDZZcT4) - Telusko (YouTube)
```